From bf8f119774dc2eabc5b9bc9f348cd95063ab2199 Mon Sep 17 00:00:00 2001
From: Lukas Slebodnik <lslebodn@redhat.com>
Date: Wed, 15 Mar 2017 12:07:37 +0100
Subject: [PATCH 112/135] SPEC: Drop conditional build for
 krb5_local_auth_plugin

It was mainly aimed for time when stable CentOS and
rhel nightly had different versions of krb5.

Anyway, rhel7.0 and rhel <= 6.6 are already out of support

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 contrib/sssd.spec.in | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/contrib/sssd.spec.in b/contrib/sssd.spec.in
index 39a974edebba3dbcd7625d1729b4a7330eaa8a27..84400e03e3fa6ead6d6cac57b3843ef79ef9ee6b 100644
--- a/contrib/sssd.spec.in
+++ b/contrib/sssd.spec.in
@@ -97,10 +97,6 @@
     %global with_cifs_utils_plugin_option --disable-cifs-idmap-plugin
 %endif
 
-%if (0%{?fedora} || (0%{?rhel} == 7 &&  0%{?rhel7_minor} >= 1) || (0%{?rhel} == 6 &&  0%{?rhel6_minor} >= 7))
-    %global with_krb5_localauth_plugin 1
-%endif
-
 %if (0%{?fedora})
     %global with_python3 1
 %else
@@ -1046,11 +1042,9 @@ done
 %dir %{_sysconfdir}/cifs-utils
 %ghost %{_sysconfdir}/cifs-utils/idmap-plugin
 %endif
-%if (0%{?with_krb5_localauth_plugin} == 1)
 %dir %{_libdir}/%{name}
 %dir %{_libdir}/%{name}/modules
 %{_libdir}/%{name}/modules/sssd_krb5_localauth_plugin.so
-%endif
 %{_mandir}/man8/pam_sss.8*
 %{_mandir}/man8/sssd_krb5_locator_plugin.8*
 
-- 
2.12.2

